### ISSUE TITLE
Veronte / Veronte / Board : Pinout configuration

### ISSUE THREAD

**Issue by gir2** [gir2/prueba_issues#30]
**Reported by admin on 5 Mar 2013 15:46 UTC**
None

**Comment by gir2** [comment]
**Comment by dbs on 28 Aug 2013 17:06 UTC**
Configuracin interfaces (e.g. RS232 como tnel; ECAP como RPM; etc.)
CAN
PWM duty cycle
GPIO

**Comment by gir2** [comment]
**Comment by dbs on 28 Aug 2013 17:24 UTC**
Configuracin Freewave a travs de puerto serie
GPIO tunel (seta roja)
GPIO desde Neurona!

**Comment by gir2** [comment]
**Comment by dbs on 31 Aug 2013 11:26 UTC**
- RS232: si se utiliza como tnel o hay algn dispositivo compatible de una lista
- RS232 radio: si se utiliza como tnel o hay alguna radio compatible de una lista (Freewave, etc.)
- CAN: si se utiliza como tnel o hay algn dispositivo compatible 

- Configuracin de los dispositivos compatibles (e.g. potencia datalink <1W)

**Comment by gir2** [comment]
**Comment by dbs on 31 Aug 2013 11:37 UTC**
De paso hay que poner un indicador de salud de perifricos/payload (cuenta de trfico?) E.g. transponder

**Comment by gir2** [comment]
**Modified by miguel.garcia on 2 Sep 2013 07:39 UTC**

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 6 Sep 2013 15:18 UTC**
In [6510eb485f8e174e8d4d4fca3d80e81967a6ee70/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="6510eb485f8e174e8d4d4fca3d80e81967a6ee70"
Refs #29. Interface configuration: GPIOs, ECAPs and SCIs
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 27 Sep 2013 10:13 UTC**
In [05b3e6d3d320316b7454a8e671a40e1605ee0269/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="05b3e6d3d320316b7454a8e671a40e1605ee0269"
Refs #29. Config Manager: Initial load in order by Id
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 27 Sep 2013 10:13 UTC**
In [09777b9eb829547fa69fc38cca563ca137964d1c/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="09777b9eb829547fa69fc38cca563ca137964d1c"
Refs #29. Pinout setup
- GPIO_if can be non static instance
- Included some namespaces in Dsp28335_ent
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 27 Sep 2013 10:13 UTC**
In [2696c35045f5102b71fbd81152f973f67cd73104/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="2696c35045f5102b71fbd81152f973f67cd73104"
Refs #29. Created list of gpiodev and added to interfaces list
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 27 Sep 2013 10:13 UTC**
In [9eac9ebb180f84b827c723eb125f7157e4d06099/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="9eac9ebb180f84b827c723eb125f7157e4d06099"
Refs #29. Added Tunnel Manager
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 27 Sep 2013 10:13 UTC**
In [7e45d7493125160a878ae368901bc37f72d61b59/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="7e45d7493125160a878ae368901bc37f72d61b59"
Refs #29. Modified Tunnel Manager
- Iport is Itunable now
- Added Tunarray of Iport pointers
- Added Gpio leds, Gpio pwms and external RS232
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 27 Sep 2013 10:13 UTC**
In [0b06e66a07c6a22945bd01a028976f401d217030/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="0b06e66a07c6a22945bd01a028976f401d217030"
Refs #29. Temporal configuration for input SCI tunnel
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 27 Sep 2013 10:13 UTC**
In [6931681df69a20dae5ce7c9eac3ce19805f2c2cb/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="6931681df69a20dae5ce7c9eac3ce19805f2c2cb"
Refs #29. Created consumers structure
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 27 Sep 2013 10:14 UTC**
In [c159cd0aac900b3c13aba5a3be45e994a90bbb0f/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="c159cd0aac900b3c13aba5a3be45e994a90bbb0f"
Refs #29. Added funcionality to SCIs
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 27 Sep 2013 10:14 UTC**
In [fdb4c6d4434844a8bea38dc05fbaf22ef7cfc609/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="fdb4c6d4434844a8bea38dc05fbaf22ef7cfc609"
Refs #29. Added funcionality to ECAPs
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 2 Oct 2013 10:49 UTC**
In [9fd0932f43a86f2221e2a6a4cd0a1e0f49440bc9/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="9fd0932f43a86f2221e2a6a4cd0a1e0f49440bc9"
Refs #29. ECAP funcionalities.
- Included Gps timepulse as funcionality
- Updted to work with peripherals without ISR
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 2 Oct 2013 10:49 UTC**
In [0872d1d7815871bcb253afa9cd9996724884d25b/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="0872d1d7815871bcb253afa9cd9996724884d25b"
Refs #29. Created GPIO tunnel
- ECAP as GPIO with tunnel funcionality
- Fixed ISR initialization bug
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 2 Oct 2013 10:49 UTC**
In [ae28d3efccc72c8037b94c8f02c299b106f75c78/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="ae28d3efccc72c8037b94c8f02c299b106f75c78"
Refs #29. SCI_if ISR configurable.
Fixed GPIO tunnel bug in Veronte Ground
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 3 Oct 2013 07:47 UTC**
In [66a0e4fc99237f64bb6cf08b69989991890dc093/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="66a0e4fc99237f64bb6cf08b69989991890dc093"
Refs #29. ECAP with Pulse Per Second funcionality (old RPM)
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 7 Oct 2013 07:22 UTC**
In [7a6b7f5d7c40ac7b7efa3c9398d033d930621c07/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="7a6b7f5d7c40ac7b7efa3c9398d033d930621c07"
Refs #29. Stick capture with new operational mode
- Local stick mode to air capture
- Remote stick mode to ground capture
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 7 Oct 2013 07:22 UTC**
In [f22e3496b09d13c3ff1d006d0c00e327e9d2d06e/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="f22e3496b09d13c3ff1d006d0c00e327e9d2d06e"
Refs #29. Port manager in ground
```

**Comment by gir2** [comment]
**Comment by miguel.garcia on 10 Oct 2013 09:08 UTC**
Falta por hacer configurables y darle funcionalidades a las siguientes interfaces:
- SPI (SD, Ethernet+Commgr, Tunel, ...)
- MCBSP (Igual que SPI)
- CAN (Tunel, ...)
- I2C (Sensores, Servos, Tuneles, ...)

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 14 Oct 2013 09:45 UTC**
In [b99e8659d6c9989a1b421f5df127abcb24b419c6/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="b99e8659d6c9989a1b421f5df127abcb24b419c6"
Refs #29, #224. Configurable SPI.
- SPI as Itunable
- MicroSD abstract communication interface (SPI or MCBSP)
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 14 Oct 2013 09:45 UTC**
In [0bef614a667e9e3921169304e8170fc7797cba69/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="0bef614a667e9e3921169304e8170fc7797cba69"
Refs #29, #224. Changes in MCBSP initialization
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 14 Oct 2013 09:45 UTC**
In [f993a7816e58f16d973569a8563ee89b1ff54489/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="f993a7816e58f16d973569a8563ee89b1ff54489"
Refs #29, #224.
- Configurable MCBSP
- Changes in SPI xmit justified
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 16 Oct 2013 10:06 UTC**
In [9816b6ec0bb3b767678deb733cc511b9ab11fbd6/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="9816b6ec0bb3b767678deb733cc511b9ab11fbd6"
Refs #29, #224. External interrupt interface created
- Instance of XINT on Utierra class
- Modified Ethport to use XINT
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 16 Oct 2013 10:06 UTC**
In [ffee30fc94cb61937ec4c492c5fa5f216020c8f2/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="ffee30fc94cb61937ec4c492c5fa5f216020c8f2"
Refs #29, #224. Split Eth_port in two classes: Eth_port and Enc424j600
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 18 Oct 2013 08:26 UTC**
In [39ad91a2a4107f6bfca6a94a05beb185563927b3/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="39ad91a2a4107f6bfca6a94a05beb185563927b3"
Refs #29, #224. Configure SPI funcionality
```

**Comment by gir2** [comment]
**Comment by miguel.garcia <miguel.garcia@embention.com> on 24 Oct 2013 07:40 UTC**
In [a1593865c509b5c719e3315860e39fdbb8da68a3/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="a1593865c509b5c719e3315860e39fdbb8da68a3"
Refs #29, #224. Created SPI configurable files (SD and Flash)
```

**Comment by gir2** [comment]
**Comment by mgg <miguel.garcia@embention.com> on 13 Dec 2013 14:16 UTC**
In [e42bdbd8815c4633c4f2582f830e23ef8bfbcb8d/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="e42bdbd8815c4633c4f2582f830e23ef8bfbcb8d"
Refs #29 #224 #247 Veronte 3
- Configurable McBSP
- Ethernet throw McBSP
- Configurable Commgr route ports
```

**Comment by gir2** [comment]
**Comment by mgg <miguel.garcia@embention.com> on 3 Jan 2014 17:27 UTC**
In [9805fd8325cf16416731c9b8a8a6685a401f9ac6/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="9805fd8325cf16416731c9b8a8a6685a401f9ac6"
Refs #29. CAN itunable
- CAN and mailboxes configurable
- Merged TX and RX mailboxes in one class
```

**Comment by gir2** [comment]
**Comment by mgg <miguel.garcia@embention.com> on 3 Jan 2014 17:27 UTC**
In [e2415cc8be371614724f9f129ca41f449d592ea6/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="e2415cc8be371614724f9f129ca41f449d592ea6"
Refs #29. CAN Itunable
- Acceptance mask enabled in reception mailbox
- Received frames added to RX Fifo in ISR, ordered by timestamp
```

**Comment by gir2** [comment]
**Comment by mgg <miguel.garcia@embention.com> on 3 Jan 2014 17:27 UTC**
In [db343ac321b7be13a193e06915c8a125aed8a3c8/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="db343ac321b7be13a193e06915c8a125aed8a3c8"
Refs #29. CAN Itunable
- Modified Can_Msg_LP to be a CanMgr Consumer
- Added CanMgr as a ComMgr port
- Protected Fifo0 equals operator for out of bounds writing
- Protected TmWire for CRC fail messages
NOTE: This commit setups CAN comunication between Ground and Air
```

**Comment by gir2** [comment]
**Comment by mgg <miguel.garcia@embention.com> on 3 Jan 2014 17:27 UTC**
In [8e051c2bba96839387539a9a4f0ad7b8f9b4dc1b/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="8e051c2bba96839387539a9a4f0ad7b8f9b4dc1b"
Refs #29. CAN Itunable.
- CAN rx is always with ISR enabled
- UTierra run high task at 100Hz
- Modified CanMgr cset/cget
```

**Comment by gir2** [comment]
**Comment by mgg <miguel.garcia@embention.com> on 10 Jan 2014 18:12 UTC**
In [4beae6b87e38f72f462a321f6c1fd41bb2665fe9/Veronte.git]:
```CommitTicketReference repository="Veronte.git" revision="4beae6b87e38f72f462a321f6c1fd41bb2665fe9"
Fix #29. Can interface configuration bug fixed
```
